plugins/node/opentelemetry-instrumentation-pg/package.json

```diff
@@ -71,7 +71,7 @@
   "dependencies": {
     "@opentelemetry/core": "^2.0.0",
     "@opentelemetry/instrumentation": "^0.202.0",
-    "@opentelemetry/semantic-conventions": "^1.27.0",
+    "@opentelemetry/semantic-conventions": "^1.34.0",
```

Comment:
Try to minimize the min version used here to the one you actually need.
I haven't looked at what semconv exports are actually used here.

Reply:
I used this one, because that is the one where we marked db as stable, so I want to use the values from the stable file, and not incubating, etc